docs/pages/concepts/logical/relationships.md

```diff
@@ -276,6 +276,9 @@ Designs are evaluated by the [Policy Engine]({{site.baseurl}}/concepts/logical/p
 
 Patches in Meshery relationships utilize strategies and references (mutatorRef/mutatedRef) for the from and to fields. These convey the property path that will be updated as the relationship is created.
 
+### Cavets and Considerations
+1. If user creates a hierarchical-inventory relationship between `pod` and any other high level kubenetes resources like `deployment` or `statefulset`. After the relationship is established. Unfortunately, there’s no system to remove the extra pod configuration automatically in this situation. This can result in duplication, where both the pod and deployment have similar configurations. It’s important to be aware of this possibility and manage configurations carefully to avoid unexpected issues during deployment
```

Comment:
```suggestion
1. If the user creates a `Hierarchical Inventory` relationship between `Pod`, `Job`, and any other high-level Kubernetes resources like `Deployment`, ' StatefulSet`, or `CronJobs`, after the relationship has been established unfortunately, there’s no system to remove the extra pod configuration automatically. 
If the design is not configured with `labels` `selectors` and `replicas` appropriately, there's a possibility of additional resources getting provisioned when deployed. eg: The relationship between a Pod and deployment can result in 2 Pods (1 pod coming as part of deployment resource) and 1 Deployment.  It’s important to be aware of this possibility and manage configurations carefully to avoid unexpected issues during deployment
```